website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md

```diff
@@ -0,0 +1,160 @@
+
+```bash
+helm uninstall appsmith
+```
+
+## Configure parameters
+
+To ensure that the Business Edition Helm chart runs, you need to make some changes to the `values.yaml` file. Follow the steps below to configure parameters:
+
+1. Add the following snippet to the bottom of your `values.yaml` file. This snippet enables PostgreSQL.
+
+   ```yaml
+   postgresql:
+     enabled: true
+     auth:
+       username: root
+       password: <PASSWORD>
+       postgresPassword: <POSTGRESQL_PASSWORD>
+       database: keycloak
+   ```
+
+2. If present, remove the highlighted lines, regardless of their value.
+
+   ```yaml
+   image:
+     # highlight-next-line
+     registry: anything
+     # highlight-next-line
+     repository: anything
+     pullPolicy: Always
+     tag: "latest"
+   ```
+
+3. Choose, or create a shared file system. For more information, see [Create a shared file system.](/getting-started/setup/installation-guides/kubernetes/business-edition#create-a-shared-file-system)
+
+4. Add the license key and a few other variables related to keycloak to `applicationConfig` section:
+
+   ```yaml
+     APPSMITH_LICENSE_KEY: ""
+     APPSMITH_KEYCLOAK_DB_DRIVER: ""
+     APPSMITH_KEYCLOAK_DB_USERNAME: ""
+     APPSMITH_KEYCLOAK_DB_PASSWORD: ""
+     APPSMITH_KEYCLOAK_DB_URL: ""
+     APPSMITH_KEYCLOAK_DB_NAME: "keycloak"
+   ```
+
+## Install Helm chart 
+
+To add and deploy the new Helm chart, run the following command:
+
+   ```bash
+   helm repo add appsmith-ee https://helm-ee.appsmith.com
+   helm repo update
+   helm install appsmith appsmith-ee/appsmith-ee -n <namespace> -f values.yaml
+   ```
+
+For more information, see [installing Business Edition with Kubernetes](/getting-started/setup/installation-guides/kubernetes/business-edition).
+
+
+## Restore backup
+
+To restore the backup, follow the below steps:
+
+1. To copy the Appsmith backup into the new Appsmith pod, run the following command:
+
+   ```bash
+   kubectl cp appsmith-backup-2022-10-24T07-09-56.930Z.tar.gz <namespace>/<pod_name>:/appsmith-stacks/data/backup/
+   ```
+
+2. To copy the keycloak backup into the new Appsmith pod, run the following command:
+
+   ```bash
+   kubectl cp keycloak_bkp.json <namespace>/<pod_name>:/appsmith-stacks/data/
+   ```
+
+3. To restore Appsmith data, run the following command:
+
+   ```bash
+   kubectl exec -it <namespace>/<pod_name> -- appsmithctl restore
+   ```
+
+   Once the data is restored, the pod restarts automatically.
+
+4. To restore the keycloak data, run the following command:
+
+   ```bash
+   kubectl exec -it <namespace>/<pod_name> -- /bin/sh /opt/keycloak/bin/standalone.sh -b 0.0.0.0 -Djboss.socket.binding.port-offset=1 -Dkeycloak.migration.action=import -Dkeycloak.migration.provider=singleFile -Dkeycloak.migration.file=/appsmith-stacks/data/keycloak_bkp.json -Dkeycloak.migration.strategy=OVERWRITE_EXISTING
+   ```
+
+   Monitor the output and look for similar lines:
+
+   ```
```

Comment:
This should ideally be a screenshot. If we can disable the copy button for such output-related content, that would be great.

Reply:
As it's easy for users to copy and search the text in the terminal. IMO we should keep it as text. We'll change the highlighting once we have the component merged.

Reply:
As mentioned above, let's keep it as is. I was wrong :)